pkg/archive/archive_unix_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"testing"
```

Comment:
Yes, it's because os.Stat().Sys() implements type syscall.Stat_t and not unix.Stat_t. The difference is unix.Stat_t doesn't have additional padding at the end. There are multiple asserts to make sure it's of type syscall.Stat_t, so those fail.

I could probably remove all of the type assertions looking for syscall.Stat_t, but I wasn't sure if that would mess up anything else and this seemed fine. I can look more into changing it if you think that's better. 🤷‍♂️